docs/format_examples_datastore.py

```diff
@@ -0,0 +1,204 @@
+import pooch
+from pathlib import Path
+
+EXAMPLES_FOLDER = Path('format_examples')
+
+format_examples = [
+    dict(
+        name='TIFF (Tagged Image File Format)',
```

Comment:
In the actual table presentable, can we make the explanation of the acronyms less prominent?  Lots of people know JPEG, PNG, DICOM, etc. and never know what they stand for; having the primary name stand out more would be better.

Reply:
Long names moved to tooltips in [544cc37](https://github.com/girder/large_image/pull/1622/commits/544cc37c7745974555ebe2d01b325cbe7d43e9d8)